<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peak&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mass&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="EPICSAppTitleWidget" name="EPICSAppTitleWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Click and drag here</string>
   </property>
   <property name="whatsThis">
    <string>The EPICSAppTitleWidget widget provides EPICS domain aware title</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(202, 223, 159);</string>
   </property>
   <property name="text">
    <string>ddnnx-VA-RGA-0x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Peak Jump Mass Selection</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>101</y>
     <width>124</width>
     <height>370</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelPeak">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_10">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_11">
        <property name="text">
         <string>11</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPeak_12">
        <property name="text">
         <string>12</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="EpicsTextEdit" name="editMass_1">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN1MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_2">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN2MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_3">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN3MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_4">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN4MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_5">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN5MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_6">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN6MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_7">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN7MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_8">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN8MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_9">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN9MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_10">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN10MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_11">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN11MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EpicsTextEdit" name="editMass_12">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Click and drag here</string>
        </property>
        <property name="whatsThis">
         <string>The EpicsTextEdit widget provides EPICS aware text edit</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="pv">
         <string>FE02I-VA-RGA-01:PJ:BIN12MASS</string>
        </property>
        <property name="pvfield" stdset="0">
         <string>The specific record name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnApply">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>480</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EPICSAppTitleWidget</class>
   <extends>QLabel</extends>
   <header>epics_app_title_widget</header>
  </customwidget>
  <customwidget>
   <class>EpicsTextEdit</class>
   <extends>QWidget</extends>
   <header>epicsedittextwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
